### PR TITLE
reciprocal_ 

### LINKED ISSUE #22297: reciprocal_
To-do list: #15115

### PR DESCRIPTION
**Pull request by samthakur587**
# added the reciprocal_ to paddle tensor frontend

## In this PR i have added the reciprocal_ to paddle frontend tensor by calling the reciprocal itself from the the tensor class. passing all the backend tests.

image

Close #22297

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -747,3 +747,8 @@ def remainder(self, y, name=None):
 
     def is_floating_point(self):
         return paddle_frontend.is_floating_point(self._ivy_array)
+
+    @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
```
Comment (HaiderSultanArc) [inline ivy/functional/frontends/paddle/tensor/tensor.py:751]: Hey @samthakur587 . There's a conflict here. It's a simple one. Can you resolve it please! 😄

Comment (samthakur587) [reply ivy/functional/frontends/paddle/tensor/tensor.py:785]: Hi! @HaiderSultanArc i resolved the conflict. now its ready to merge. 🚀

Comment (HaiderSultanArc) [reply ivy/functional/frontends/paddle/tensor/tensor.py:785]: Hey @samthakur587 your tests are failing now. Can you have a look? Only look for your failing tests. You can ignore others. Thanks 😄

Comment (samthakur587) [reply ivy/functional/frontends/paddle/tensor/tensor.py:785]: Hi! the failing test is not related to my PR. what's happen i thing someone has made changes to testing pipeline. in local its showing this error.

image

I don't know what happening but its start after i pull the latest changes from ivy. 😄

Comment (HaiderSultanArc) [reply ivy/functional/frontends/paddle/tensor/tensor.py:785]: @sherry30 🤔 do you know something about this?

**Comment by samthakur587** [comment]
Hi! @HaiderSultanArc Now i have pull the latest changes its passing all the tests. 😄 hope now its ready to merge. 🚀 

image


thanks 
sam

**Pull request merged**